### PR TITLE
Copy changes to support appeals launch

### PR DESCRIPTION
**Pull request by aub**
Note: We do not wish to merge this until the appeals status feature is ready for launch.

These are copy changes to support the appeals launch for department-of-veterans-affairs/vets.gov-team#3680. It pretty much just changes "claims" to "claims and appeals" in a number of places.

Closes department-of-veterans-affairs/vets.gov-team#3680
Closes https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3266
Closes https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3706
Closes https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3729

### CODE REVIEW

**Review by bshyong — APPROVED** [review]
Copy changes LGTM. 

Noting that there are some other changes we need, either in this PR or a different one,  notably 
1. moving the new copy for appeals from the `disability-claims-beta` to replace the content in the current `disability-claims` folder
2. Small changes to the error handling logic/copy to account for appeals data https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3266

**Comment by aub** [comment]
@bshyong I think I understood from our meeting just now that you're going to work on the two things you mentioned above and push the changes into this PR. Is that correct? If there's anything I can or should help with on them please let me know.

**Comment by bshyong** [comment]
@aub yes, I've edited the description accordingly to close the issue for error handling. Content changes and error handling commits are now in this PR

**Comment by aub** [comment]
I just pushed a few more changes here to address department-of-veterans-affairs/vets.gov-team#3729 in [this commit](https://github.com/department-of-veterans-affairs/vets-website/pull/5960/commits/71a89cc92f49a5505a7dce1ccd1976dbe523a8fd). @U-DON, would you have a minute to review them? It's mostly straightforward, except that I have a bad feeling that my CSS changes could be not great, or at least not standard with how we do things.

**Comment by aub** [comment]
Also, @U-DON do you have thoughts about how to address the codeclimate issues being raised above with duplicate code?

**Comment by leannammiller** [comment]
for @U-DON and @bshyong to review and merge.

File: content/pages/disability-benefits/claims-appeal.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ When the Regional Office (RO) receives your Notice of Disagreement (NOD), you'll
 <a class="usa-button-primary" href="/track-claims">Track your appeal</a>
 </div>
 
-### Appeals Process Overview
+<h3 class="no-margin">Appeals Process Overview</h3>
```
Comment (U-DON) [inline content/pages/disability-benefits/claims-appeal.md]: Unless there's a specific style here that's overriding it, I think `h3` already has `margin: 0` set in `_va.scss`.

File: src/sass/base/_va.scss [hunk 2]
```diff
@@ -376,6 +376,10 @@ h6 {
   }
 }
 
+h3.no-margin {
+  margin: 0 0 0 0;
```
Comment (U-DON) [inline src/sass/base/_va.scss]: If this is really necessary, `margin: 0` could be a shorthand for this.

File: src/sass/base/_va.scss [hunk 3]
```diff
@@ -1440,7 +1444,11 @@ p.info-message {
   list-style-type: none;
 }
 
-.vertical-list-group .list-group-item{
+.vertical-list-group.more-bottom-cushion {
```
Comment (U-DON) [inline src/sass/base/_va.scss]: Is this class actually being used anywhere?

**Review by U-DON — APPROVED** [review]
LGTM. Noticed some minor things.

As for codeclimate, we tend not to pay attention to it since it can pick up some pretty insignificant things. In this case, it seems like it's all about duplicate code in the beta folders, which doesn't seem like a big deal, and is probably expected.

File: src/sass/base/_va.scss (hunk 3 above)
Comment (aub) [inline src/sass/base/_va.scss]: Whoops, somehow I must have rolled back the change where I used it. Added now.

**Comment by bshyong** [comment]
merge conflicts are resolved

**Pull request merged**